playbooks/roles/grafana/tasks/firewall.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Firewalld enablement
+
+- name: Enable firewalld
```

Comment:
let's please replace this with firewalld_extra_ports_enable (similarly to https://github.com/opentelekomcloud-infra/system-config/blob/main/inventory/service/group_vars/alerta.yaml#L21) and get rid of firewall management completely

Reply:
done